### PR TITLE
#9283: fix prerelease browser workflow flakiness

### LINKED ISSUE #9283: Fix Pre-release Browser workflow
## Describe the bug

Follow-up of https://github.com/pixiebrix/pixiebrix-extension/issues/9193

Our Pre-release Browser workflow is failing consistently despite improvements to test flakiness in CI. https://github.com/pixiebrix/pixiebrix-extension/actions/workflows/e2e-test-pre-release-browsers.yml

### PR DESCRIPTION
**Pull request by fungairino**
## What does this PR do?

- Closes #9283

## Demo

- See the latest test run in the Actions tab for this repository: https://github.com/pixiebrix/pixiebrix-extension/actions/runs/11448241104

## Checklist

- [x] Added jest or playwright tests and/or storybook stories

For more information on our expectations for the PR process, see the
[code review principles doc](https://www.notion.so/pixiebrix/Code-Review-Principles-1ce7276b82a84d2a995d55ad85e1310d?pvs=4)

### CODE REVIEW

File: end-to-end-tests/pageObjects/extensionConsole/workshop/workshopPage.ts [hunk 1]
```diff
@@ -22,24 +22,19 @@ import { CreateWorkshopModPage } from "./createWorkshopModPage";
...
   }
 
   async goto() {
-    await this.page.goto(this.extensionConsoleUrl);
-    await this.getByRole("link", {
-      name: "Workshop",
-    }).click();
-    await this.getByRole("heading", { name: "Workshop" }).waitFor();
-    await this.getByRole("cell").first().waitFor();
+    await this.page.goto(this.workshopUrl);
```
Comment (fungairino) [inline end-to-end-tests/pageObjects/extensionConsole/workshop/workshopPage.ts:37]: The purpose of this change is to load the workshop page faster, and avoid any possible flakiness from the /api/bricks api request being cancelled in between navigating from the root mods page to the workshop page.

File: end-to-end-tests/pageObjects/extensionsShortcutsPage.ts [hunk 2]
```diff
@@ -116,7 +116,9 @@ export class ExtensionsShortcutsPage extends BasePageObject {
 
     if (isChrome(this.chromiumChannel)) {
       await expect(
-        this.getByLabel(/Shortcut Toggle Quick Bar for PixieBrix/),
+        this.getByRole("textbox", {
+          name: "Shortcut Toggle Quick Bar for PixieBrix",
+        }),
```
Comment (fungairino) [inline end-to-end-tests/pageObjects/extensionsShortcutsPage.ts:121]: The previous locator was running into an error where there was more than one element matching the locator query. 

nit: could be refactored into a locator variable.

File: end-to-end-tests/tests/extensionConsole/activation.spec.ts [hunk 3]
```diff
@@ -252,30 +251,22 @@ test("can activate a mod via url", async ({
...
-      if (!activationPage) {
-        throw new Error("Extension console page not found");
-      }
-    }).toPass({ timeout: 20_000 });
-  } else {
-    await activationPage.waitForURL(
-      `chrome-extension://${extensionId}/options.html#/marketplace/activate/${modIdUrlEncoded}`,
-    );
-  }
+  // Browsers sometimes throw this error if we use `waitForURL` for this redirect behavior:
```
Comment (fungairino) [inline end-to-end-tests/tests/extensionConsole/activation.spec.ts:254]: saw this behavior for chrome today as well, so I'm just switching this over now.

File: end-to-end-tests/tests/pageEditor/liveEditing.spec.ts [hunk 4]
```diff
@@ -33,7 +33,7 @@ test("live editing behavior", async ({
     await activationPage.goto();
     await activationPage.clickActivateAndWaitForModsPageRedirect();
 
-    await page.goto("/advanced-fields/");
+    await page.goto("/");
```
Comment (fungairino) [inline end-to-end-tests/tests/pageEditor/liveEditing.spec.ts:36]: the "advanced-fields" sub page for some reason does not trigger the load event when under heavy traffic. This is most likely due to the heavy rich text editor assets being throttled / delayed.

Comment (fungairino) [reply end-to-end-tests/tests/pageEditor/liveEditing.spec.ts:36]: See for example this test run error:
https://github.com/pixiebrix/pixiebrix-extension/actions/runs/11447557916/job/31849153012

File: end-to-end-tests/pageObjects/extensionConsole/workshop/workshopPage.ts (hunk 1 above)
Comment (fungairino) [reply end-to-end-tests/pageObjects/extensionConsole/workshop/workshopPage.ts:37]: I am seeing more flakiness in the latest test run so I might need to revert this particular change.

**Review by mnholtz — APPROVED** [review]

**Pull request merged**